zero/src/prover.rs

```diff
@@ -28,6 +32,11 @@ use crate::fs::generate_block_proof_file_name;
 use crate::ops;
 use crate::proof_types::GeneratedBlockProof;
 
+pub struct ProofRuntime {
```

Comment:
Could be nice to have some high-level explanation of this struct, i.e. its purpose to distinguish base STARK proof runtimes vs aggregation ones.

Reply:
fixed